ts/session/snode_api/swarmPolling.ts

```diff
@@ -62,9 +62,9 @@ export class SwarmPolling {
 
   public removePubkey(pubkey: PubKey) {
     if (this.pubkeys.indexOf(pubkey) !== -1) {
```

Comment:
This will actually still allow duplicate `PubKey` to be present as it's an object.
A better solution might be
```js
this.pubkeys = this.pubkeys.filter(key => !pubKey.isEqual(key));
this.groupPubkeys = this.groupPubkeys.filter(key => !pubKey.isEqual(key));
```

Reply:
updated